vocabularies/georesources-report-types.ttl

```diff
@@ -382,6 +411,17 @@ grt:petroleum-report-infrastructure a skos:Concept ;
     skos:prefLabel "Petroleum Report - Infrastructure"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/georesource-report> .
 
+grt:petroleum-report-relinquishment a skos:Concept ;
+    skos:altLabel "Final Relinquishment (ATP)"@en,
+        "Final Relinquishment (PL)"@en,
+        "Petroleum Report - Final Relinquishment"@en,
+        "Petroleum Report - Relinquishment"@en ;
+    skos:definition "A report that is required to be lodged with a partial of full surrender of a Petroleum Lease or Authority to Prospect, detailing the work undertaken on the permit throughout the life of tenure."@en  ;

```

Comment:
partial of full surrender - think this is meant to be "or"
is this meant to say "relinquishment" instead of surrender?
if so, then "throughout the life of tenure" would change.

Reply:
```suggestion
    skos:definition "A report that is required to be lodged with a partial or full relinquishment of a Petroleum Lease or Authority to Prospect, detailing the work undertaken on the relinquished portion of the permit throughout the life of tenure."@en  ;

```